### DISCUSSION TITLE
Response variable data in Jspsych.data stored as "array" in phpmyadmin

### DISCUSSION THREAD

**Discussion by dm0rg8n** [jspsych/jsPsych#2020] [Q&A]
Hi,

I am currently saving my data using the scripts provided here https://github.com/vassar-cogscilab/php-mysql-save-data. After a recent update to jspsych the scripts (https://github.com/jspsych/jsPsych/pull/1523) now appear to store data in the 'response' key value pair of jspsych.data as the word "array" in phpmyadmin and not the responses provided by the participants, e.g., {consent1: yes, consent2: yes, and so on (see attached image). The script worked as expected when we used the previous version of jspsych, we are now only encountering this problem with the updated version. 

![image](https://user-images.githubusercontent.com/56552238/127496557-d581dc9f-3d2b-49b6-9f35-b0a8b6d19f97.png)

I think this might be because "jsPsych data now stores JavaScript arrays and objects, rather than JSON-stringified versions of objects/arrays" as per one of the recent updates. Would anyone know how to resolve this issue?

Sorry if I have missed the solution to this in the disucssion and thank you in advance for any help you can provide, I really appreciate it.

**Comment by Nega100** [comment]
Hi @dm0rg8n  add the JSON.stringify to your plugin or on finish section.
```var get_data = {
"rt": rt,
"response":JSON.stringify(response)
}
```

**Comment by rariel1** [comment]
I'm having the same issue. Did you find a solution? Previous experiments that use to save correctly are now just saving "response" data with the word "array".
